### PR TITLE
Remove "(?)" from comments in pkg/devicemapper

### PR DESCRIPTION
**Pull request by vdemeester**
Got merged with it, removing it as it doesn't add anything 🐷. 

This was bugging me :sweat_smile:, so Vincent The First (@vbatts), I need your benediction that those comments are alright :wink:.

🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

### CODE REVIEW

File: pkg/devicemapper/devmapper_wrapper.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ const (
 	LoNameSize       = C.LO_NAME_SIZE
 )
 
-// DeviceMapper Udev consts. (?)
+// DeviceMapper Udev consts.
```
Comment (vbatts) [inline pkg/devicemapper/devmapper_wrapper.go]: "Devicemapper cookie flags"

File: pkg/devicemapper/devmapper.go [hunk 2]
```diff
@@ -309,7 +309,7 @@ func FindLoopDeviceFor(file *os.File) *os.File {
 	return nil
 }
 
-// UdevWait wakes any processes that are waiting for udev to complete the specified cookie. (?)
+// UdevWait wakes any processes that are waiting for udev to complete the specified cookie.
```
Comment (vbatts) [inline pkg/devicemapper/devmapper.go]: s/wakes/waits/

**Comment by vbatts** [comment]
couple of nits

File: pkg/devicemapper/devmapper.go (hunk 2 above)
Comment (vdemeester) [inline pkg/devicemapper/devmapper.go]: @vbatts that would give "UdevWait waits for any processes that are waiting for udev to complete the specified cookie." ? 

ETOOMUCHWAIT :sweat_smile:

Comment (vbatts) [inline pkg/devicemapper/devmapper.go]: It's more that the kernel may have jobs for the specified cookie and this
call will block until they complete

Comment (vdemeester) [inline pkg/devicemapper/devmapper.go]: Oh I see :)

**Comment by vdemeester** [comment]
@vbatts updated it :wink:

**Comment by vbatts** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**